api/v1/controllers/categories.js

```diff
@@ -0,0 +1,53 @@
+import { toInt } from '../helpers/functions';
+
+// Import data structure for categories
+import { allCategories, Category } from '../models/Category';
+
+export default {
+
+  getAllCategories(req, res) {
+    res.status(200).json(allCategories);
+  },
+
+  getCategoryById(req, res) {
+    const categoryId = toInt(req.params.categoryId);
+    if (!categoryId) {
+      res.status(422).send({ errors: { categoryId: 'Category Id is required' } });
+    }
+
+    const category = allCategories.find(item => item.id === categoryId);
+    res.status(200).json(category);
+  },
+
+  createCategory(req, res) {
+    const category = new Category(req.body);
+    allCategories.push(category);
+
+    res.status(200).json(allCategories);
```

Comment:
Don't know if I gave this feedback yesterday, but for methods that handle POST and PUT, prefer to return the object that was just created or modified in the response, instead of returning the array that has all the items

Reply:
Yeah. You gave the feedback. I'll update it accordingly